README.md

```diff
@@ -139,10 +141,10 @@ To test the WebSocket connection and ensure that everything is set up correctly,
 Replace `<YOUR_API_KEY>` with your actual API key and run:
 
 ```bash
-wscat -s 'graphql-ws' -H 'x-api-key: <YOUR_API_KEY>' -c wss://api.hgraph.dev/v1/graphql
+wscat -s 'graphql-ws' -H 'x-api-key: <YOUR_API_KEY>' -c wss://api.hgraph.io/v1/graphql
```

Comment:
```suggestion
wscat -s 'graphql-ws' -H 'x-api-key: <YOUR_API_KEY>' -c wss://testnet.hedera.api.hgraph.dev/v1/graphql
```

api.hgraph.<io|dev> i deprecated to serve hedera data and may be a network aggregation endpoint instead before...so lets just use the endpoint formats as describe here - https://docs.hgraph.com/graphql-api/endpoints and https://docs.hgraph.com/rest-api/endpoints